### DISCUSSION TITLE
✅ 🔗 Loop Carousel: Center mode (Duplicate of #21195)

### DISCUSSION THREAD

**Discussion by mateitudor** [elementor/elementor#21453] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Great job team on the loop carousel 🖤. You made my day with this update!

I have quite a straightforward suggestion, it would be amazing if the loop carousel would have a "center mode".
UI pattern: te first and the last item in the visible row is partially cut out, signaling that there is more content to be scrolled.

### Describe the solution you'd like

Here's an example:
![Crude loop carousel UI example](https://ucf63174b4a014a540d1db932cf4.previews.dropboxusercontent.com/p/thumb/AB2IR9h8s1D7QwFI9YqdjTiNLIYg1duBUIIcQC2gz4sXli1Wy3kmTHE_UmrwKcs4I93xUarCG2dWOwOSbZ0d2zvthRN-h-2MtOV4e9Mma1f-Gdqiz9HwREUmChPTJ3qVESZZNnaKgK9c4RdUlLZ_0Meu30Y_iQC48ZP26smhcCOmy8uPKKmQUs-6aqEZ8IH5lceuAaX5JYLotShiqmRonTbowxqi5Op8um04MnHeRHdOnQ2LYOm3V8A7Gg0pOzlgxsVjqjkA2XDKqsDbaruFYCXpq9Hx185ZFJhzSYaFCOUxvzwS5ow5gb7W5VCk2pDSdSHWrG6mpZCZXm5u0EdjtksOR5XyuXj0cIdEpynKU4af9QkeGGTwuJCs0O72G5JDNxHWuJlkVlsBPJ9rUsnXKPnwm5I5hmwjRnYSc_1OjUfnCQ/p.png)

Thank you in advance.

**Comment by vladninja** [comment]
Duplicate: https://github.com/orgs/elementor/discussions/21195

**Comment by nicholaszein** [comment]
Hello there!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with Elementor Pro v3.15! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
